deployment/v3/utils/postgres-backup/install.sh

```
@@ -0,0 +1,115 @@
+fi
+
+NS=postgres-backup
+CHART_VERSION=0.0.1-develop
+
+echo Create $NS namespace
+kubectl create ns $NS
+
+function installing_postgres-backup() {
+  helm repo update
+
+  read -p "Please provide S3 Bucket Name: " S3_BUCKET
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)